common/src/main/java/com/revenuecat/purchases/common/BillingWrapper.kt

```diff
@@ -134,23 +138,26 @@ class BillingWrapper(
         onReceiveSkuDetails: (List<SkuDetails>) -> Unit,
         onError: (PurchasesError) -> Unit
     ) {
-        debugLog("Requesting products with identifiers: ${skuList.joinToString()}")
+        log(LogIntent.DEBUG_INFO, OfferingStrings.FETCHING_PRODUCTS.format(skuList.joinToString()))
         executeRequestOnUIThread { connectionError ->
             if (connectionError == null) {
                 val params = SkuDetailsParams.newBuilder()
                     .setType(itemType).setSkusList(skuList).build()
                 withConnectedClient {
                     querySkuDetailsAsync(params) { billingResult, skuDetailsList ->
                         if (billingResult.responseCode == BillingClient.BillingResponseCode.OK) {
-                            debugLog("Products request finished for ${skuList.joinToString()}")
-                            debugLog("Retrieved skuDetailsList: ${skuDetailsList?.joinToString { it.toString() }}")
+                            log(LogIntent.DEBUG_INFO, OfferingStrings.FINISH_FETCHING_PRODUCTS
```

Comment:
I really think that the tense should be a part of the name for these strings, otherwise there's no telling apart stuff like "finishing" and "finished"